src/jobs/models/database/sequelize/sequelizeConnector.js

```diff
@@ -68,7 +68,7 @@ async function getJobsAndParse(jobId) {
 
     allJobs.forEach(job => {
         job.emails = job.emails && job.emails.length > 0 ? job.emails.map(sqlJob => sqlJob.dataValues.address) : undefined;
-        job.webhooks = job.webhooks && job.webhooks.length > 0 ? job.webhooks.map(sqlJob => sqlJob.dataValues.url) : undefined;
+        // job.webhooks = job.webhooks && job.webhooks.length > 0 ? job.webhooks.map(sqlJob => sqlJob.dataValues.url) : undefined;
```

Comment:
uncomment, should return string array of uuid

Reply:
done